html-forms/html-forms_personal-profile-form/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>Personal Profile Form</title>
+    <link rel="stylesheet" href="./css/styles.css" />
+  </head>
+
+  <header>
+    <h1>Create your Personal Profile</h1>
+  </header>
+
+  <body>
+    <form aria-labelledby="title" aria-describedby="description">
+
+    <p id="description">
+          We want to get to know you! Please enter your personal information.
+      </p>
+     
+
+        <fieldset>
+          <legend>Personal Information</legend>
+          <label for="input-name">First Name:</label>
+          <input id="input-name" type="text" minlength="4" required />
+          <label for="input-name">Last Name:</label>
+          <input id="input-name" type="text" minlength="4" required />
```

Comment:
Your first two input fields have exactly the same id. Please, see my comment above regarding unique ids.